components/admin_console/billing/billing_subscriptions/limits.scss

```diff
@@ -0,0 +1,48 @@
+@import 'utils/variables';
+@import './mixins';
+
+.ProductLimitsPanel {
+    @include cloud-card;
+
+    &__title {
+        @include cloud-card-title;
```

Comment:
Opted for mixins here because:
* Only some of the styles are reshared, but enough that we want to easily keep the same look and feel
* We didn't want the components to share the same class
* These are a local mixin to this part of the webapp, so not worried about proliferating the use of mixins.